tasks/deploy-hashed-assets.js

```diff
@@ -37,7 +38,8 @@ function upload(params) {
 			});
 }
 
-function task (app) {
+function task (opts) {
```

Comment:
think this is ok, couldn't find any apps that were calling this directly with an app name